tests/lightning_local_backend/run.sh

```diff
@@ -97,6 +97,9 @@ for ckpt in mysql file; do
   grep -Fq "No table has lost intermediate files according to given config" $TEST_DIR/lightning_ctl.output
 
   # when position of chunk file doesn't equal to offset, intermediate file should exist
+  run_sql 'DROP DATABASE IF EXISTS cpeng;'
```

Comment:
Why do clean up here since checkpoint is enabled

Reply:
this pr make https://github.com/pingcap/tidb-lightning/pull/493/files#diff-ca1e86afc37a9b71183fb70c7fd2d65ea68a24a60e55a664d2f379be63fc7f9dR2336
the fail point doing nothing actually.
so the first run will succeed and the checkpoint is removed.

<img width="1001" alt="Screen Shot 2021-04-07 at 7 28 38 PM" src="https://user-images.githubusercontent.com/1681864/113859425-76185100-97d7-11eb-9fc4-5e62f3910c49.png">

Reply:
if not remove it will fail to check the empty table since it's a new import.
I have no clue why the failpoint is changed to doing nothing so just keep clean up here like before now.